backend/src/users/guard/isMe.guard.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  CanActivate,
+  ExecutionContext,
+  Injectable,
+  UnauthorizedException
+} from '@nestjs/common';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class IsMeGuard implements CanActivate {
```

Comment:
Couldn't we find a better name ?
Like `IsCUrrentUser`

Reply:
Yes we can